charts/argo-rollouts/README.md.gotmpl

```diff
@@ -78,10 +78,10 @@ If dashboard is installed by `--set dashboard.enabled=true`, checkout the argo-r
 
 ### To 1.0.0
 
-* This is a breaking change which only supports Helm v3.0.0+ now. If you still use Helm v2, please consider upgrading because v2 is EOL since November 2020.   
+* This is a breaking change which only supports Helm v3.0.0+ now. If you still use Helm v2, please consider upgrading because v2 is EOL since November 2020.
```

Comment:
Why this change? I assume that 2 trailing spaces are markdown newlines

Reply:
Nope.  Just spaces.  My local editor keeps auto-trimming on save so my commit gets out of whack with generated.  Hence the removal from `README.md.gotmpl`.  Put back still or can remove.

I am also tempted to remove the last line from the README so I don't have to include a README edit every time I update the version of this chart.  Not a fan.  wdyt?

Reply:
Okay 👍 i think we should introduce a .editorconfig in the repos root dir ;)

Yup, you could replace it with `main` instead of the version. As a downside users are always redirected to the newest values.yaml file. But thats how all other charts are working.

Reply:
Btw: it looks different ;) 
![grafik](https://user-images.githubusercontent.com/7290987/193152942-d403eebc-09c1-464c-954f-5aefba97807b.png)

Reply:
Going to put the spaces back in.  I'm also getting a markdown preview plugin for vscode 😄

Reply:
🤣🤣

Reply:
spaces back, editorconfig being respected.  awwwwyeah